lib/ui/shared/bottom_sheets/profile_botton_sheet/custom_user_bottom_sheet_viewmodel.dart

```diff
@@ -1,18 +1,32 @@
+import 'package:hng/services/current_user_profile.dart';
+import 'package:hng/services/local_storage_services.dart';
+import 'package:hng/utilities/storage_keys.dart';
 import 'package:stacked/stacked.dart';
 import 'package:stacked_services/stacked_services.dart';
 
 import '../../../../app/app.locator.dart';
 import '../../../../app/app.router.dart';
 import '../../../../models/user_model.dart';
-import '../../../../package/base/server-request/api/http_api.dart';
-import '../../../../services/local_storage_services.dart';
-import '../../../../utilities/constants.dart';
-import '../../../../utilities/storage_keys.dart';
 
-class CustomUserBottomSheetViewModel extends FutureViewModel {
+final storageService = locator<SharedPreferenceLocalStorage>();
```

Comment:
move this into the class

Reply:
> move this into the class

Causes an error: "The instance member 'storageService' can't be accessed in an initializer."